packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -47,19 +49,23 @@ export async function handleGetBlindedMessageSig(
   request: Request<{}, {}, GetBlindedMessageSigRequest>,
   response: Response
 ) {
+  const logger: Logger = response.locals.logger
   try {
     if (!isValidGetSignatureInput(request.body)) {
-      respondWithError(response, 400, WarningMessage.INVALID_INPUT)
+      Counters.responses.labels('400').inc()
+      respondWithError(response, 400, WarningMessage.INVALID_INPUT, logger)
       return
     }
-    if (!(await authenticateUser(request, getContractKit()))) {
-      respondWithError(response, 401, WarningMessage.UNAUTHENTICATED_USER)
+    if (!(await authenticateUser(request, getContractKit(), logger))) {
+      Counters.responses.labels('401').inc()
+      respondWithError(response, 401, WarningMessage.UNAUTHENTICATED_USER, logger)
       return
     }
     logger.debug('Requesting signatures')
     await requestSignatures(request, response)
   } catch (e) {
-    respondWithError(response, 500, ErrorMessage.UNKNOWN_ERROR)
+    Counters.responses.labels('500').inc()
+    respondWithError(response, 500, ErrorMessage.UNKNOWN_ERROR, logger)
```

Comment:
What do you think about moving the counting into `respondWithError`?

Reply:
Good idea